tools/jdbc/src/test/java/org/duckdb/test/TestDuckDBJDBC.java

```diff
@@ -2159,14 +2160,36 @@ public static void test_unsiged_integers() throws Exception {
 		conn.close();
 	}
 
+
+	public static void test_get_schema() throws Exception {
+		DuckDBConnection conn = (DuckDBConnection) DriverManager.getConnection("jdbc:duckdb:");
+
+		assertEquals(conn.getSchema(), "main");
```

Comment:
Could we also add a test where we change the schema, and verify that `getSchema` correctly returns the modified schema?

Reply:
Done!